cluster/gce/configure-vm.sh

```diff
@@ -439,6 +443,7 @@ clusters:
 - name: local
   cluster:
     certificate-authority-data: ${KUBELET_CA_CERT}
+    server: {{ api_servers }}
```

Comment:
This addition is a bit confusing because the kubelet actually ignores this part of the kubeconfig file.